alto/commands/cromwell/run.py

```diff
@@ -195,6 +195,8 @@ def submit_to_cromwell(
     if time_out is not None:
         status = wait_and_check(server, port, resp_dict["id"], time_out)
         print(f"{{\"job_id\": \"{resp_dict['id']}\", \"status\": \"{status}\"}}")
+    if resp.status_code == 201:
```

Comment:
Does it make sense to merge the `if-else` block in Line 188 -- 192 here?

Reply:
I cleaned up the code a bit